src/Microsoft.DotNet.Build.Tasks/PackageFiles/versioning.targets

```diff
@@ -41,8 +41,7 @@
   </PropertyGroup>
 
   <Target Name="DecideIfWeNeedToIncludeDllSafeSearchPathAttribute" 
-    Condition="'$(IsDotNetFrameworkProductAssembly)' == 'true' AND '$(IsTestProject)' != 'true'"
-    BeforeTargets="GenerateAssemblyInfo">
+    Condition="'$(IsDotNetFrameworkProductAssembly)' == 'true' AND '$(IsTestProject)' != 'true'">
```

Comment:
Do we not need this guy any longer? If not then we should probably remove it since I believe that with your change we are basically removing it from the build.

Reply:
We do still need it.  I just switched how it is invoked.  Instead of being `BeforeTargets`, I made the other targets Depend on this.  With the new SDK, it doesn't make sense for this to be `BeforeTargets=GenerateAssemblyInfo`.